packages/landing/app/layout.tsx

```diff
@@ -0,0 +1,27 @@
+import { Navbar } from "@/ui/navbar";
+import "./globals.css";
+import type { Metadata } from "next";
+
+const DEFAULT_STACKLY_URL = "https://stackly.eth.limo";
+
+export const metadata: Metadata = {
+  metadataBase: new URL(process.env.STACKLY_URL ?? DEFAULT_STACKLY_URL),
+  title: "Stackly | Stack crypto over time.",
+  description:
+    "Stackly is a simple, non-custodial tool that uses the CoW protocol to place recurring swaps based on DCA",
+};
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="en">
+      <body className="font-sans bg-surface-25 text-em-high">
+        <Navbar />
```

Comment:
Navbar can't be part of UI, it will be different in the landing page

Reply:
yeah, I'll move it to `components`